scripts/setup_imdb.py

```diff
@@ -35,7 +35,7 @@ def is_setup():
 assert len(sys.argv) == 2
 table_dir = sys.argv[1]
 
-LOCATION = "https://archive.org/download/imdb_20200624/imdb.zip"
+LOCATION = "http://www.dropbox.com/s/fgju7cifpehqlww/imdb.zip?dl=1"
 FILE_NAME = "imdb.zip"
```

Comment:
This must be the original source then: https://github.com/gregrahn/join-order-benchmark/commits/master

I'd add this as a comment for reference.

Reply:
Good idea.